map_model/src/map.rs

```diff
@@ -1016,14 +1014,15 @@ fn make_half_map(
             map.intersections[src_i.0].outgoing_lanes.push(id);
             map.intersections[dst_i.0].incoming_lanes.push(id);
 
-            let (unshifted_pts, other_lanes_width): (PolyLine, Distance) = if lane.reverse_pts {
```

Comment:
The changes here and below are great, thank you! This is a nice way to deal with the fwd/back distinction